test/SparqlAntlrParserTest.cpp

```diff
@@ -936,9 +934,14 @@ TEST(SparqlParser, Query) {
   auto expectQuery = ExpectCompleteParse<&Parser::query>{
       {{INTERNAL_PREDICATE_PREFIX_NAME, INTERNAL_PREDICATE_PREFIX_IRI}}};
   auto expectQueryFails = ExpectParseFails<&Parser::query>{};
+  // TODO: re-add simple tests that also assert the ParsedQuery.
```

Comment:
This todo is only still relevant for `CONSTRUCT` clauses and should already be fixed and then removed in this PR.